frontend/packages/dev-console/console-extensions.json

```diff
@@ -1 +1,151 @@
-[]
+[
+  {
+    "type": "dev-console.add/action",
+    "properties": {
+      "id": "import-from-samples",
+      "url": "/samples",
+      // t('devconsole~Samples')
```

Comment:
I'd suggest to wait until #8363 is merged and then remove these `// t('..')` comments in JSON files.

Reply:
Removed the comment and rebased with https://github.com/openshift/console/pull/8363.